lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -366,7 +366,8 @@ async sub transform_transaction {
         );
 
         my @transactions = await $self->_check_contract_transaction($transaction, $receipt);
-        push @transactions, $transaction;
+        $amount = $amount->bstr();
```

Comment:
```suggestion
```

Reply:
@reginaldo-binary What is the suggested change ? Currently doing the same `$amount = $amount->bstr();`